samples/apps/forum/src/types/ccf.ts

```diff
@@ -48,6 +48,26 @@ export interface KVMap {
 
 export type KVMaps = { [key: string]: KVMap };
 
+export interface ProofElement {
+  left?: string
+  right?: string
+}
+
+export type Proof = ProofElement[];
+
+export interface Receipt {
+  signature: string;
```

Comment:
I haven't implemented this yet, but, there's the choice of providing `ArrayBuffer`s, or Base64/hex-encoded strings. I'm leaning slightly towards `ArrayBuffer` to avoid having an opinion on final encoding. It would require a bit more work from the app developer, but nothing too dramatic.